Felienne/hedy
# [BUG FIX] Re-direct on teacher invite link is working again

**Description**
In this PR we fix an issue with the teacher invite link re-direct. Due to the user being set as a teacher but not being updated in the session it seems they weren't a teacher until logging out and logging in again. We fix this by updating the session value as wel on a valid invite link. This way the re-direct works again and the welcome message is displayed as expected.


**Fixes**
This PR fixes #2566.

**How to test**
This one is quite complex to test as you need to add the environment variable `TEACHER_INVITE_CODE` to your IDE. Then make sure you have a non-teacher account and navigate to `/invite/<your code>`. Verify that you are correctly re-directed to the for-teachers page with a welcome message pop-up.
